recipes/refinitiv-data/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "refinitiv-data" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/refinitiv-data-{{ version }}.tar.gz
+  sha256: c4ad0698a9ed2bdf11af836faeb7ea797e72edc048df043301307f1e1b980d88
```

Apparently the license was only included with 1.1.0:
```suggestion
{% set version = "1.1.0" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/refinitiv-data-{{ version }}.tar.gz
  sha256: 471a0decd1e16d70c25084f33439a0038ba65a99c15e571fd55d03c766aceb41
```

---

done